tests/sentry/manager/tests.py

```diff
@@ -20,16 +20,25 @@ def test_add_tags(self):
         environment = self.create_environment()
 
         with self.tasks():
-            Group.objects.add_tags(group, environment, tags=(
-                ('foo', 'bar'), ('foo', 'baz'), ('biz', 'boz')))
+            Group.objects.add_tags(
+                group,
+                environment,
+                tags=[
+                    ('foo', 'bar'),
+                    ('foo', 'baz'),
```

Comment:
… did you know we supported multiple tag values per key for an event?

Reply:
I did not. Is that why we use lists of tuples instead of a dict? Snuba is definitely not set up to support these in any case.